src/odemis/acq/acqmng.py

```diff
@@ -83,6 +84,197 @@ def acquire(streams, settings_obs=None):
     return future
 
 
+def acquireZStack(streams, zlevels, settings_obs=None):
```

Comment:
I think the docstring format should include some empty lines. You are using the reStructuredText format here  (my favorite). Can you follow that format here?

https://queirozf.com/entries/python-docstrings-reference-examples